src/main/java/org/advent/daySeven/Hand.java

```diff
@@ -0,0 +1,174 @@
+package org.advent.daySeven;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Hand {
+    String handString;
+    List<Character> relevantCards;
+    List<Character> restOfHand;
+    int bid;
+    Type type;
```

Comment:
default visibility should only be used when you explicitly need it

Reply:
made privat final